libs/platform/experience/src/services/layout/plugins/properties/index.ts

```diff
@@ -1,4 +1,5 @@
 export * from './bleed';
 export * from './divider';
 export * from './overlap';
+export * from './shadow';
```

Comment:
Just wondering. Does just setting of shadow be called a layout?

Reply:
agreed, over engineered. There was initially more to it, but it's simplified and remains like this afaics.